{{cookiecutter.project_name}}/monitoring/README.md

```diff
@@ -0,0 +1,63 @@
+# Setup Model Monitoring
+
+This directory contains setup scripts intended to generate notebook that can be uploaded to databricks workspace and execute
+to set up model monitoring.
+
+## Prerequisites
+
+### Unity Catalog
+The inference table to be monitored must exist in Unity Catalog.
```

Comment:
Nit: skip "inference"? Or use "table with model predictions" to be more clear?

Reply:
+1